packages/block-editor/src/components/colors-gradients/control.js

```diff
@@ -137,22 +129,46 @@ function ColorGradientControlInner( {
 						</legend>
 					) }
 					{ canChooseAColor && canChooseAGradient && (
-						<TabPanel
-							className="block-editor-color-gradient-control__tabs"
-							tabs={ TABS_SETTINGS }
-							initialTabName={
-								gradientValue
-									? TAB_GRADIENT.value
-									: !! canChooseAColor && TAB_COLOR.value
-							}
-						>
-							{ ( tab ) => renderPanelType( tab.value ) }
-						</TabPanel>
+						<div className="block-editor-color-gradient-control__tabs">
```

Comment:
Looking at the `block-editor-color-gradient-control__tabs` classname, it looks like it's only used once, [to set some padding on the gradient panel](https://github.com/WordPress/gutenberg/blob/00d1226c654626070f2e60655706c03c715d459d/packages/block-editor/src/components/colors-gradients/style.scss#L18-L22).

Also, inspecting the target element, it looks like the styles in question are already overridden by some other styles:

![Screenshot 2023-11-29 at 16 21 35](https://github.com/WordPress/gutenberg/assets/1083581/e577b145-6d04-42b8-bb35-10bd4fd3cddb)

All this to say that we can probably safely remove both this `div`, and the styles associated to the `block-editor-color-gradient-control__tabs` classname

Reply:
Good catch, thanks!

This does introduce a small visual change: 4 extra pixels of vertical space between the tabs and the tabpanels. It's coming from the parent `VStack`'s `spacing={ 1 }`. The wrapping `div` that was in place previously (even on `trunk`) was preventing that. We can easily remove that prop if we want to keep the spacing as it was, or leave it in depending on what looks best.

Reply:
Alright, let's add back the div, although no classname is needed